### PR TITLE
fixes #11923 - Updates version of hammer-cli-foreman in gemspec

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by domcleal** [comment]
Please note that as usual we need a release as soon as possible to include this change, as we're already packaging hammer_cli 0.4.0 since it was released a couple of days ago.  If this is going to be a problem every quarter, please don't pin it so tightly.

**Comment by mmoll** [comment]
Per semver `~> 0.4` should be fine

File: hammer_cli_foreman_tasks.gemspec [hunk 1]
```diff
@@ -20,5 +20,5 @@ DESC
   s.files = Dir["{lib,config}/**/*", "LICENSE", "README.md"]
 
   s.add_dependency "powerbar", "~> 1.0.11"
-  s.add_dependency "hammer_cli_foreman", "> 0.1.1", "< 0.4.0"
+  s.add_dependency "hammer_cli_foreman", "~> 0.4"
```
Comment (domcleal) [inline hammer_cli_foreman_tasks.gemspec]: This increases the minimum to 0.4, which is probably not necessary as there hasn't been a code change to warrant it.  I'd suggest just keeping the old style but changing the second part to `< 1.0.0`.

**Comment by mmoll** [comment]
:+1:

**Pull request merged**